08_files_lab/02.py

```diff
@@ -1,26 +1,17 @@
-"""
-Write a program that takes 3 names. The first two
-are existing file names and the last is a new file name/
-The program should write the lines from the first two 
-files interwinded into the output file. So if we have
+#!/usr/bin/python
 
-a.txt with
-a1
-a2
+import sys
+from itertools import izip_longest
 
-and b.txt with
-b1
-b2
-
-then running
-
-python 02.py a.txt b.txt c.txt
-will result in the creation of c.txt with the following content:
-
-a1
-b1
-a2
-b2
-
-"""
+file1 = sys.argv[1]
+file2 = sys.argv[2]
+file3 = sys.argv[3]
```

Comment:
Alternatively use:

```
_, file1, file2, file3 = sys.argv
```

The first underscore is actually a variable that will take sys.argv[0]. By convention we use underscore as a variable name that holds values nobody cares about